change/@ni-nimble-components-4ace02ad-9e11-459a-b1f0-dce6436e251e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Reminder to make similar changes to spright configuration if you haven't already.